tfx/examples/cifar10/cifar10_pipeline_native_keras.py

```diff
@@ -0,0 +1,205 @@
+  """Implements the CIFAR10 image classification pipeline using TFX."""
+  input_config = example_gen_pb2.Input(splits=[
+      example_gen_pb2.Input.Split(name='train', pattern='train/*'),
+      example_gen_pb2.Input.Split(name='eval', pattern='test/*')])
+
+  examples = external_input(data_root)
+
+  # Brings data into the pipeline.
+  example_gen = ImportExampleGen(input=examples, input_config=input_config)
+
+  # Computes statistics over data for visualization and example validation.
+  statistics_gen = StatisticsGen(examples=example_gen.outputs['examples'])
+
+  # Generates schema based on statistics files.
+  schema_gen = SchemaGen(
+      statistics=statistics_gen.outputs['statistics'], infer_feature_shape=True)
+
+  # Performs anomaly detection based on statistics and data schema.
+  example_validator = ExampleValidator(
+      statistics=statistics_gen.outputs['statistics'],
+      schema=schema_gen.outputs['schema'])
+
+  # Performs .ations and feature engineering in training and serving.
+  transform = Transform(
+      examples=example_gen.outputs['examples'],
+      schema=schema_gen.outputs['schema'],
+      module_file=module_file)
+
+  # We take the raw examples as the input to the trainer,
+  # as data augmentation needs to be applied on the fly.
+  # When traning on the whole dataset, use 20000 for train steps,
+  # 156 for val steps
+  def _create_trainer(module_file, instance_name):
+    return Trainer(
+        module_file=module_file,
+        custom_executor_spec=executor_spec.ExecutorClassSpec(GenericExecutor),
+        examples=transform.outputs['transformed_examples'],
+        transform_graph=transform.outputs['transform_graph'],
+        schema=schema_gen.outputs['schema'],
+        train_args=trainer_pb2.TrainArgs(num_steps=72),
+        eval_args=trainer_pb2.EvalArgs(num_steps=3),
+        instance_name=instance_name)
+
+  # Uses user-provided Python function that trains a Keras model.
+  trainer = _create_trainer(module_file, 'cifar10')
+
+  # Get the latest blessed model for model validation.
+  model_resolver = ResolverNode(
+      instance_name='latest_blessed_model_resolver',
+      resolver_class=latest_blessed_model_resolver.LatestBlessedModelResolver,
+      model=Channel(type=Model),
+      model_blessing=Channel(type=ModelBlessing))
+
+  # Uses TFMA to compute an evaluation statistics over features of a model and
+  # performs quality validation of a candidate model.
+  eval_config = tfma.EvalConfig(
+      model_specs=[tfma.ModelSpec(label_key='label_xf', model_type='tf_lite')],
+      slicing_specs=[tfma.SlicingSpec()],
+      metrics_specs=[
+          tfma.MetricsSpec(metrics=[
+              tfma.MetricConfig(
+                  class_name='SparseCategoricalAccuracy',
+                  threshold=tfma.config.MetricThreshold(
+                      value_threshold=tfma.GenericValueThreshold(
+                          lower_bound={'value': 0.3}),
```

Comment:
this is really low. Can we find a way to make this higher with the small dataset that is included? Perhaps adjust the training params. If you need to adjust training params, then specify the ones that we would like to use for the full example in the readme.